roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -15,22 +15,37 @@ class DockerImageAvailability(OpenShiftCheck):
 
     skopeo_image = "openshift/openshift-ansible"
 
-    # FIXME(juanvallejo): we should consider other possible values of
-    # `deployment_type` (the key here). See
-    # https://github.com/openshift/openshift-ansible/blob/8e26f8c/roles/openshift_repos/vars/main.yml#L7
     docker_image_base = {
         "origin": {
             "repo": "openshift",
             "image": "origin",
         },
+        "online": {
```

Comment:
> We could remove online altogether from this PR and just treat it as an invalid deployment type if you think this would be the better approach.

Yes, let's remove it. We don't need to try to support something that is not supported by the installer. Thanks for investigating.

BTW - if it is not valid any longer, perhaps we should remove it from https://github.com/openshift/openshift-ansible/blob/6051550cb/roles/openshift_sanitize_inventory/vars/main.yml#L7? (/cc @sosiouxme )